<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addFriend</class>
 <widget class="QDialog" name="addFriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arkadaş Ekle</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>42</y>
     <width>83</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kullanıcı Adı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>75</y>
     <width>50</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sunucu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="friend_uname">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>37</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="server">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>200</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addFriendExit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFriendSubmit">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekle</string>
   </property>
  </widget>
  <widget class="QLabel" name="addFriendSymbol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../addFriend1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Statics/group.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
